public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -187,16 +180,7 @@ export class PanelChrome extends Component<Props, State> {
   // The next is outside a react synthetic event so setState is not batched
   // So in this context we can only do a single call to setState
   onDataUpdate(data: PanelData) {
-    const { isInView, dashboard, panel, plugin } = this.props;
-
-    if (!isInView) {
-      if (data.state !== LoadingState.Streaming) {
-        // Ignore events when not visible.
-        // The call will be repeated when the panel comes into view
-        this.setState({ refreshWhenInView: true });
-      }
-      return;
-    }
```

Comment:
removed this condition as it's a bit strange, it sort of throws away the data and sets refreshWhenInView to true if a panel get's data when out of view. We already stop new refreshes when out of view so, feels unnecessary to throw away the data here to then have to query for it again when the user scrolls back